app/views/event_emails/new.html.haml

```diff
@@ -5,7 +5,7 @@
       Select recipients
       %span.caret
     %ul#state-menu.dropdown-menu{"aria-labelledby" => "select_recp"}
-      - %w[All Accepted Incomplete Submitted Approved Cancel].each do |s|
+      - %w[All Accepted Incomplete Submitted Approved Canceled].each do |s|
```

Comment:
I would bring this in some order, either logical (incomplete -> submitted -> approved -> accepted -> canceled -> all) or in alphabetical order.